Hazelcast.Net/Hazelcast.Client.Spi/ClientInvocationService.cs

```diff
@@ -269,10 +269,17 @@ private void HandleInvocationException(ClientInvocation invocation, Exception ex
                     }
                 }
                 //Fail with exception
-                if (!invocation.Future.IsComplete)
+                try
                 {
                     invocation.Future.Exception = exception;
                 }
+                catch (InvalidOperationException e)
+                {
+                    if (Logger.IsFinestEnabled())
```

Comment:
Maybe we should change Java side to debug 🤔 . 
There is nothing for customer can do with this log as it is only a debug message that should be reported back to us.